python/tests/conftest.py

```diff
@@ -255,16 +258,27 @@ def lmdb_version_store_prune_previous(version_store_factory):
 
 @pytest.fixture
 def lmdb_version_store_big_map(version_store_factory):
-    return version_store_factory(lmdb_config={"map_size": 2**30})
+    return version_store_factory(lmdb_config={"map_size": 2 ** 30})
+
 
-@pytest.fixture(scope='function', params=[False])
```

Comment:
Wait, where did these get changed to single quotes in the first place?

Reply:
Oops. I have run black formatting with this file. Will remove the last commit.